### DISCUSSION TITLE
Entering String and Integer values

### DISCUSSION THREAD

**Discussion by asif-rashid** [osiegmar/FastCSV#62] [Q&A] [answered]
I couldn't find any example, where my row data have multiple data types. It looks like writeRow is only accepting String array.

**Comment by osiegmar** [comment, marked as answer]
FastCSV is a fairly low-level CSV implementation that simply implements CSV as defined per [RFC-4180](https://datatracker.ietf.org/doc/html/rfc4180) in a robust and high performant manner. FastCSV does not perform any data type mapping or conversion on its own, as this is not the intended purpose. If you want to write Integer values you have to convert them to String (e.g. by using `java.lang.Integer#toString()` or `java.lang.Integer#toString(int)`) first.
